File: myonlinefm/src/main/java/com/sec/myonlinefm/classificationprogram/data/RequestProgramClassify.java
package com.sec.myonlinefm.classificationprogram.data;

import java.io.Serializable;

/**
 * Created by SRC-TJ-MM-BinYang on 2018/4/13.
 */

public class RequestProgramClassify implements Serializable {
    private int id = 0;
    private String name = null;
    private int sectionId = 0;

    public void setId(int id) {
        this.id = id;
    }

    public int getId () {
        return id;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getName() {
        return name;
    }

    public void setSectionId(int sectionId) {
        this.sectionId = sectionId;
    }

    public int getSectionId() {
        return sectionId;
    }
}

File: myonlinefm/src/main/java/com/sec/myonlinefm/classificationprogram/data/RequestProgramClassify.kt
package com.sec.myonlinefm.classificationprogram.data

import java.io.Serializable

/**
 * Created by SRC-TJ-MM-BinYang on 2018/4/13.
 */
class RequestProgramClassify : Serializable {
    internal var id = 0
    private var name: String? = null
    internal var sectionId = 0
    fun setId(id: Int) {
        this.id = id
    }

    fun getId(): Int {
        return id
    }

    fun setName(name: String?) {
        this.name = name
    }

    fun getName(): String? {
        return name
    }

    fun setSectionId(sectionId: Int) {
        this.sectionId = sectionId
    }

    fun getSectionId(): Int {
        return sectionId
    }
}